packages/react-mongo/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Looks like this is coming here from another PR.

Reply:
oh, hmmm

Reply:
left over iteration - I removed it.

Reply:
Npm packages are required for tests to run. Node modules should never be present during publication, but we need to keep the package.json files.

A better solution to prevent publishing node_modules is .meteorignore, but I don't know if that works in packages.